src/compositor/lipstickcompositorwindow.cpp

```diff
@@ -337,7 +348,6 @@ void LipstickCompositorWindow::handleTouchCancel()
         inputDevice->sendTouchCancelEvent();
         inputDevice->setMouseFocus(0, QPointF());
     }
-    window()->removeEventFilter(this);
```

Comment:
This seems dangerous to me. Are we sure an item is always going to be removed from the scene when it is hidden/disabled?